src/openqaoa-core/tests/test_workflows.py

```diff
@@ -1726,6 +1731,640 @@ def test_change_properties_after_compilation(self):
+        Based on the input device.
+        Also Checks if defaults from workflows are used in the backend.
+        """
+
+        num_assets, budget = 5, 3
+        mu, sigma, _ = generate_random_portfolio_data(num_assets = num_assets, num_days = 15, seed = 1)
+        problem = PortfolioOptimization(mu, sigma, risk_factor = None, budget = budget, penalty = None)
+
+        fqaoa = FQAOA()
+        fqaoa.set_device(create_device(location="local", name="vectorized"))
+        fqaoa.fermi_compile(problem=problem.qubo, n_fermions=3)
+
+        orbitals = get_analytical_fermi_orbitals(n_qubits=num_assets, n_fermions=budget, lattice="cyclic", hopping=1.0)
+        initial_state = get_statevector(orbitals)
+
+        self.assertEqual(type(fqaoa.backend), QAOAvectorizedBackendSimulator)
+
+        self.assertEqual(fqaoa.backend.init_hadamard, False)
+        self.assertTrue(np.array_equal(fqaoa.backend.prepend_state, initial_state))
+        self.assertEqual(fqaoa.backend.append_state, None)
+        self.assertEqual(fqaoa.backend.cvar_alpha, 1)
+
+        self.assertRaises(AttributeError, lambda: fqaoa.backend.n_shots)
+
+    def test_set_backend_properties_check_backend_vectorized_w_custom(self):
+        """
+        Check if the backend returned by set_backend_properties is correct
+        Based on the input device.
+        Uses custom values for attributes in backend_properties and checks if the
+        backend object responds appropriately.
+        """
+
+        num_assets, budget = 5, 3
+        mu, sigma, _ = generate_random_portfolio_data(num_assets = num_assets, num_days = 15, seed = 1)
+        qubo_problem = PortfolioOptimization(mu, sigma, risk_factor = None, budget = budget, penalty = None).qubo
+
+        fqaoa = FQAOA()
+        fqaoa.set_device(create_device(location="local", name="vectorized"))
+
+        update_pairings = {
+            "n_shots": 10,
+            "cvar_alpha": 1,
+        }
+
+        fqaoa.set_backend_properties(**update_pairings)
+
+        fqaoa.fermi_compile(problem=qubo_problem, n_fermions=budget)
+        
+        orbitals = get_analytical_fermi_orbitals(n_qubits=num_assets, n_fermions=budget, lattice="cyclic", hopping=1.0)
+        initial_state = get_statevector(orbitals)
+
+        self.assertEqual(type(fqaoa.backend), QAOAvectorizedBackendSimulator)
+        
+        self.assertTrue(np.array_equal(fqaoa.backend.prepend_state, initial_state))
+        self.assertEqual(fqaoa.backend.cvar_alpha, 1)
+
+        self.assertRaises(AttributeError, lambda: fqaoa.backend.n_shots)
+        
+    def test_set_classical_optimizer_defaults(self):
+        pass
+
+    def test_set_classical_optimizer_jac_hess_casing(self):
+        pass
+
+    def test_set_classical_optimizer_method_selectors(self):
+        pass
+
+    def test_set_header(self):
+        pass
+
+    def test_set_exp_tags(self):
+        pass
+
+    def test_qaoa_evaluate_circuit(self):
```

Comment:
Let's rename this to `test_fqaoa_evaluate_circuit`

Reply:
Rename `test_fqaoa_evaluate_circuit`.